velox/docs/functions/presto/json.rst

```diff
@@ -133,6 +133,7 @@ JSON Functions
         SELECT json_extract(json, '$.store.book');
 
     Current implementation supports limited subset of JSONPath syntax.
+    JSON string can be incomplete.
```

Comment:
Would you include some examples to help illustrate what "incomplete" means?

Reply:
Maybe have a more clear definition, something like "JSON string can be just a prefix of a valid JSON".